app/src/composables/use-flows.ts

```diff
@@ -265,5 +272,6 @@ export function useFlows(options: UseFlowsOptions) {
 export function injectRunManualFlow() {
```

Comment:
Let’s use it with use for consistency.

```suggestion
export function useInjectRunManualFlow() {
```